Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:alpine
+MAINTAINER birgir.stefansson@schibsted.com
```

Comment:
We might want to point maintainer at the FIAAS service user (fiaas@googlegroups.com), I'm not sure... ?

Reply:
Good point since codacy was complaining about it I just added me but I'll update.

Reply:
Done